scripts/content_checks/goals.py

```diff
@@ -30,6 +30,7 @@ def check_file(filename: str, goal_names: List[str]) -> None:
             continue
         if filename.startswith('#'):
             print(f'Skipping: {filename}')
+            pass
         else:
-            print(f'Goals check: {filename}')
+            #print(f'Goals check: {filename}')
```

Comment:
should this just be removed instead of commented out?

Reply:
Yes- thanks!